### PR TITLE
fix(CX-1379): “cancel” button doesn’t clear terms and results

### PR DESCRIPTION
**Pull request by JenqaaaBY**
The type of this PR is: fix

This PR resolves [CX-1379]

### Description

This PR fixes the behavior of the cancel button. Now when you press the "cancel" button, text inputs and search results are cleared. Also in this PR, the component was refactored by using a prettier.
https://user-images.githubusercontent.com/30176348/117949037-63cf8c80-b31a-11eb-98a6-f5d50e875939.mov

### PR Checklist (tick all before merging)

- [x]  I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-1379]: https://artsyproduct.atlassian.net/browse/CX-1379

### CODE REVIEW

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Congratulations on your first contribution to Eigen 🎉. Looks good to me except few minor remarks.
- The first one is about the PR title. Ideally, it should be **fix(CX-1379):  “cancel” button doesn’t clear terms and results** (the ticket between brackets is not mandatory). This helps us have a clean git history with a unified format (see https://github.com/artsy/eigen/commits/master)
- The second one is about the last item on the PR checklist, the changelog update. I think you forgot to add it. We try to keep track of all the changes we do for every version in this changelog file https://github.com/artsy/eigen/blob/master/CHANGELOG.yml that we use also to generate release notes when we submit the app for review.

**Comment by JenqaaaBY** [comment]
> Congratulations on your first contribution to Eigen 🎉. Looks good to me except few minor remarks.
> 
> * The first one is about the PR title. Ideally, it should be **fix(CX-1379):  “cancel” button doesn’t clear terms and results** (the ticket between brackets is not mandatory). This helps us have a clean git history with a unified format (see https://github.com/artsy/eigen/commits/master)
> * The second one is about the last item on the PR checklist, the changelog update. I think you forgot to add it. We try to keep track of all the changes we do for every version in this changelog file https://github.com/artsy/eigen/blob/master/CHANGELOG.yml that we use also to generate release notes when we submit the app for review.

Thanks for your review.
I've changed the PR name and added my changes to the CHANGELOG.yml file

File: CHANGELOG.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ upcoming:
     - Add staging support for deep links on android - mounir, ole
     - Refactor filter event payloads to send JSON - roop
     - Fix issue with ARSpinner being registered more than once - mounir, kizito
+    - Fix cancel button doesnt clear terms and results - yauheni
```
Comment (MounirDhahri) [inline CHANGELOG.yml:11]: **thought:** I think this is not needed here since you already included it in the `user_facing` section

**Comment by MounirDhahri** [comment]
Great, thanks for making the changes. I left a minor comment about an extra addition to the changelog. Other than that, this is clear to go. Also there seem to be a changelog conflict

**Comment by MounirDhahri** [comment]
Awesome thank you
#squashongreen

**Pull request merged**